metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -82,11 +91,27 @@
     OwnershipClass,
     OwnershipTypeClass,
 )
+from datahub.utilities.source_helpers import (
+    auto_stale_entity_removal,
+    auto_status_aspect,
+)
 
 logger = logging.getLogger(__name__)
 
 
-class LookerDashboardSourceConfig(LookerAPIConfig, LookerCommonConfig):
+class LookerStatefulIngestionConfig(StatefulStaleMetadataRemovalConfig):
```

Comment:
the docs are wrong here - you can use `StatefulStaleMetadataRemovalConfig` directly below, and delete `LookerStatefulIngestionConfig` entirely

Reply:
Oh, so is there no need to override `_entity_types` according to source ?

Reply:
That's right - I've also opened a PR to remove it elsewhere in the codebase https://github.com/datahub-project/datahub/pull/6454